assets/dns/daemonset.yaml

```diff
@@ -90,10 +90,16 @@ spec:
             declare -A svc_ips
             for svc in "${services[@]}"; do
               # Fetch service IP from cluster dns if present
-              ips=($(dig @"${NAMESERVER}" +short "${svc}.${CLUSTER_DOMAIN}"))
+              ips=($(dig -t A @"${NAMESERVER}" +short "${svc}.${CLUSTER_DOMAIN}"))
```

Comment:
How about `dig "@$NAMESERVER" +short A "${svc}.${CLUSTER_DOMAIN}" AAAA "${svc}.${CLUSTER_DOMAIN}"`?

Reply:
Although that prints an extra line: ";; Warning, extra type option", which we would need to ignore (or maybe we shouldn't ignore it...).

Reply:
This should avoid the warning: `dig "@$NAMESERVER" +short "${svc}.${CLUSTER_DOMAIN}" AAAA "${svc}.${CLUSTER_DOMAIN}"`.

Reply:
Refactor the whole thing to Go in a followup?